game_eggs/steamcmd_servers/v_rising/egg-v-rising.json

```diff
@@ -0,0 +1,231 @@
+            "container": "ghcr.io\/pterodactyl\/installers:debian",
+            "entrypoint": "\/bin\/bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "[REQUIRED] Server Query Port",
+            "description": "UDP port for Steam server list features.",
+            "env_variable": "QUERY_PORT",
+            "default_value": "9877",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|integer|between:1024,65536"
+        },
+        {
+            "name": "Automatic Updates",
+            "description": "Quickly checks for any server updates on startup, and updates if necessary.",
+            "env_variable": "AUTO_UPDATE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "boolean"
+        },
+        {
+            "name": "Game Mode",
+            "description": "What preset of game settings the server should run. Some settings may not apply after the save file is first created. Leave empty\/null if you are using a single-player uploaded save or a custom ServerGameSettings.json file.",
+            "env_variable": "GAME_SETTINGS_PRESET",
+            "default_value": "StandardPvP",
```

Comment:
default_value is StandardPvP and the rules specify StandardPVP, meaning the default does not work out of the box. Not sure which is correct, but the server has started with StandardPVP as the setting.

Reply:
> default_value is StandardPvP and the rules specify StandardPVP, meaning the default does not work out of the box. Not sure which is correct, but the server has started with StandardPVP as the setting.

He start, but after a clean install it's "". As i wrote above. Not everyone want to use presents. Presents should be optional

Reply:
Commit [bd77d26](https://github.com/parkervcp/eggs/pull/1680/commits/bd77d26939a8b18887771d0d52fb0d4164996c93) should resolve this issue. I was mistaken that `StandardPvP.json` was identical to the default `ServerGameSettings.json` file, so the default value has been changed to `null`, which should load the default `ServerGameSettings.json` file (unless a custom `ServerGameSettings.json` file is created in `~/save-data/Settings`). A custom `ServerGameSettings.json` file is not created during the installation script in case users would like to use a game settings preset (a `ServerGameSettings.json` file being present would override the preset setting).